modules/ScanningEnumeration/0x02-WebCrawling/photon.py

```diff
@@ -25,7 +25,7 @@
 
 info = "Incredibly fast crawler designed for OSINT"
 searchinfo = "Incredibly fast crawler designed for OSINT"
-properties = {"ROOT":["Root URL the crawling starts with", " "], "ARGS":["The arguments passed to Photon (formatted as after $photon -u ROOT )", " "]}
+properties = {"ROOT":["Root URL the crawling starts with", " "], "ARGS":["The arguments passed to Photon (formatted as after $photon pROOT )", " "]}
```

Comment:
I'm not quite sure what is the issue here, can you please point me out what I'm missing? :)

Reply:
Oops. I've probably hit my keyboard accidentally and didn't notice those changes.